### PR TITLE
added slack notification

### PR DESCRIPTION
**Pull request by dereban25**
Created a slack notification, added trigger on release.

### CODE REVIEW

File: .github/workflows/slack-notification-release.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: slack-notification-release
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+on:
+  release:
+    branches:
+       - main
+    types:
```
Comment (alonashkenazi) [inline .github/workflows/slack-notification-release.yaml:8]: why we need this condition?

File: .github/workflows/slack-notification-release.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        id: slackuser
+        run:
+           echo "::set-output name=username::$(grep ${{ github.event.pusher.email }} ${{ secrets.SLACK_FILE }} | awk '{print $3}')"
+      - name: Check slack
+        uses: act10ns/slack@v1
+        env:
+          user: ${{ steps.slackuser.outputs.username }}
+        with:
+           status: ${{ job.status }}
+           channel: '@${{ env.user }}'
```
Comment (alonashkenazi) [inline .github/workflows/slack-notification-release.yaml:34]: release need to be on cloudrail-dev

**Review by alonashkenazi — CHANGES_REQUESTED** [review]

File: .github/workflows/slack-notification-release.yaml (hunk 1 above)
Comment (dereban25) [reply .github/workflows/slack-notification-release.yaml:8]: Trigger only "main" branch.I can remove branch and set types "created". If failure send notification

File: .github/workflows/slack-notification-release.yaml (hunk 2 above)
Comment (dereban25) [reply .github/workflows/slack-notification-release.yaml:34]: I will change

**Review by alonashkenazi — APPROVED** [review]

**Pull request merged**